libraries/namespaces.raml

```diff
@@ -0,0 +1,7 @@
+#%RAML 1.0 Library
+annotationTypes:
+  namespace:
+    description: |
+      this annotation type allows you to associate unique identifier with a RAML document.
+    type: string
+(namespace): http://raml.org/vocabularies/annotations/core
```

Comment:
I have some doubts about the value for the annotation:

- In general it could be any unique string, but that's difficult to enforce.
- We could try to specify that it should be a URL, but that mean we are enforcing the idea of RAMLs as linked data. This is not something clear right now. For example, Exchange doesn't share it.
- It could be a URI, as the (http://raml.org/vocabularies/annotations/core or exchange:group.asset.version)
- It could be a URI prefix, like (http://raml.org/vocabularies/annotations/core#) 

Should we specific about this?
Is there a way of being more specific than just saying `type: string`?